### PR TITLE
Fix swapped commitment_cost (beta) in VQ loss computation

### LINKED ISSUE #95: maskgit_vqgan‘s vq_loss is incorrect
In the code, the vq_loss is incorrect, and the commitment_cost is placed in the wrong position.

https://github.com/bytedance/1d-tokenizer/blob/bdec006fb7226b309aaf3511956d0aafac30125a/modeling/modules/maskgit_vqgan.py#L296-L299

After correction:
In the code, the vq_loss is fixed, and the commitment_cost is placed in the correct position.

 ```
if return_loss:
    loss = self.commitment_cost  * torch.mean((z_q.detach() - hidden_states) ** 2) +  torch.mean(
        (z_q - hidden_states.detach()) ** 2
    )
```

### PR DESCRIPTION
**Pull request by Mr-Neutr0n**
## Summary

Fixes #95

The `commitment_cost` (beta) in `VectorQuantizer.forward()` was applied to the wrong loss term. Per the VQ-VAE paper ([arXiv:1711.00937](https://arxiv.org/abs/1711.00937), equation 4), the total VQ loss is:

```
L = ||sg(z_e) - e||^2 + beta * ||z_e - sg(e)||^2
```

Where:
- **Codebook loss**: `||sg(z_e) - e||^2` — trains the codebook embeddings toward the encoder outputs (no beta weighting)
- **Commitment loss**: `beta * ||z_e - sg(e)||^2` — encourages the encoder to commit to codebook embeddings (weighted by beta)

In the existing code, `hidden_states` is the encoder output (`z_e`) and `z_q` is the quantized embedding (`e`):

```python
# Before (incorrect): beta applied to codebook loss
loss = torch.mean((z_q.detach() - hidden_states) ** 2) + self.commitment_cost * torch.mean(
    (z_q - hidden_states.detach()) ** 2
)
```

```python
# After (correct): beta applied to commitment loss
loss = self.commitment_cost * torch.mean((z_q.detach() - hidden_states) ** 2) + torch.mean(
    (z_q - hidden_states.detach()) ** 2
)
```

## Test plan

- [x] Verified the fix matches the standard VQ-VAE loss formulation from the paper (equation 4)
- [x] Confirmed the change is a single-line swap with no other side effects

### CODE REVIEW

**Comment by Mr-Neutr0n** [comment]
Closing this one to tidy up my open pull requests.

It's been open around five months with no review activity, and I'd rather withdraw it than leave it sitting in your queue. Nothing needed from you, and no hard feelings at all.

If the fix is still wanted, this can be reopened, or I'm happy to redo it properly against current main. Apologies for the noise.
